docs/en/ingest-management/release-notes/release-notes-7.13.asciidoc

```diff
@@ -33,7 +33,7 @@ impact to your application.
 
 [discrete]
 [[breaking-97206]]
-.Remove Elastic Agent routes and related services
+.Remove Elastic Agent routes and related services from {kib}
```

Comment:
```suggestion
.Remove Elastic Agent routes and related services from {kib} and implement them in {fleet-server}
```

Reply:
I love the clarification - if you think it isn't quite right to extend it a bit further it is ok.  I am just putting the suggestion out there in case the extension feels right.  what do you think? I could go either way, but in this case the slight more detail may help even a bit more.  up to you

Reply:
Yah, I like your change because it's more descriptive.